src/codeu/chat/client/commandline/Chat.java

```diff
@@ -178,7 +176,7 @@ public void invoke(List<String> args) {
 		panel.register("u-add", new Panel.Command() {
 			@Override
 			public void invoke(List<String> args) {
-				final String name = !args.isEmpty() ? args.get(0).trim() : "";
+				final String name = !args.isEmpty() ? String.join(" ", args).trim() : "";
```

Comment:
Nice. Only thing I would say is maybe add a comment above these lines explaining what it does (or more accurately, *why* it's doing that).

Do we have tests for these things that need to change?